src/recommendations/tests/RecommendationsPage.test.jsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import { getConfig } from '@edx/frontend-platform';
+import { injectIntl, IntlProvider } from '@edx/frontend-platform/i18n';
+import { mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import configureStore from 'redux-mock-store';
+
+import { DEFAULT_REDIRECT_URL } from '../../data/constants';
+import * as getPersonalizedRecommendations from '../data/service';
+import RecommendationsPage from '../RecommendationsPage';
+
+const IntlRecommendationsPage = injectIntl(RecommendationsPage);
+const mockStore = configureStore();
+
+jest.mock('../data/service');
+
+describe('RecommendationsPageTests', () => {
+  let defaultProps = {};
+  let store = {};
+
+  const mockedResponse = [
```

Comment:
can you put this mocked constant data into a separate file such as `tests/constants.js`. This approach is used by learner home and I think its a good approach to separate constants from tests.